client/src/components/MultiSelectBox.js

```diff
@@ -33,6 +53,21 @@ const MultiSelectBox = () => {
         placeholder={"Wähle aus"}
         className="multiselect"
       />
+      {filteredRestaurants?.map((filteredRestaurant) => (
+        <Card
+          key={filteredRestaurant._id}
+          restaurantImg={filteredRestaurant.img}
+          restaurantName={filteredRestaurant.name}
+          restaurantDiet={filteredRestaurant.diet}
+          restaurantCuisine={filteredRestaurant.cuisine}
+        />
```

Comment:
Why don't you just pass the restaurant object into here? Would make your code much slimmer
```suggestion
        <Card
          restaurant={filteredRestaurant}
        />
```

Reply:
Thank you very much for the review! i'll get to it during my refactoring 🦾 I have it on my toDo list